plugins/zabbix.go

```diff
@@ -139,7 +141,7 @@ func collectInstanceCounters(data map[string]interface{}, counters map[string]in
 		if strings.HasPrefix(key, "c_") {
 			cList, found := value.([]interface{})
 			if !found {
-				log.Printf("Problem collecting counters, expected a list but got: " + reflect.TypeOf(value).Name())
+				log.Printf("Problem collecting counters, expected a list but got: %T", value)
```

Comment:
How fatal is this condition? If it affects the caller, you should return an error instead of just logging it.

Reply:
Not fatal. This will only skip broken metrics if any are encountered.